Libs/MRML/Core/vtkMRMLColorNode.h

```diff
@@ -110,10 +110,15 @@ class VTK_MRML_EXPORT vtkMRMLColorNode : public vtkMRMLStorableNode
       TypeModifiedEvent = 20002
     };
 
-  ///
   /// Get the 0th based nth name of this colour
```

Comment:
I know it was not your change, but this description does not make sense. Something like this would be more clear:
/// Get name of a color from its index (index is 0-based)